src/main/java/com/minecolonies/coremod/research/costs/ListItemCost.java

```diff
@@ -82,6 +90,29 @@ public void write(@NotNull final CompoundTag compound)
         compound.putInt(TAG_COST_COUNT, this.count);
     }
 
+    @Override
+    public void serialize(final @NotNull FriendlyByteBuf buf)
+    {
+        buf.writeInt(this.count);
+        buf.writeInt(this.items.size());
+        for (final Item item : this.items)
+        {
+            buf.writeItem(new ItemStack(item));
```

Comment:
since items are also in a registry we could just write their ID right, similar to how you did it for the costtype